src/components/MainComponent.js

```diff
@@ -17,6 +17,48 @@ import Debug from 'components/Debug'
 import { isChromeExtensionBackground } from 'services/notificationService'
 import Notifications from 'components/Extension/Notifications'
 
+// import * from 'pluginsTemp/index'
+// // introducing unnecessary duplication to determine what needs extraction.
+// const _ = require('underscore')
+
+// const hydratePlugin = (plugin) => {
+//   const separator = require('path').sep
+//   const directoryPath = './src/pluginsTemp'
+//   var filename = _.last(plugin.path.split(separator))
+//
+//   plugin.newPath = [directoryPath, filename].join(separator)
+//
+//   return plugin
+// }
+
+// const plugins = (environment = process.env) => {
+//   if (!environment.PLUGINS) { console.log('No plugins are defined.'); return }
+//
+//   return _.map(JSON.parse(environment.PLUGINS), hydratePlugin)
+// }
+
+//
+
+// _.each(plugins(), function (plugin) {
+//
+// })
+
+const maybePlugin = () => {
+  try {
+    const Checklist = require('components/Checklist') // eslint-disable-line no-unused-vars
```

Comment:
Does a try { } block create a new scope? I would have thought that this would need to be like

    let Checklist
    try {
       Checklist = require()
    }